dts/arm/renesas/gen3/r8a77951.dtsi

```diff
@@ -5,3 +5,30 @@
  */
 
 #include <renesas/gen3/rcar_gen3_cr7.dtsi>
+#include <dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h>
+
+&pfc {
+	can0_data_a_tx_default: can0_data_a_tx_default {
+		pins = <PIN_RD FUNC_CAN0_TX_A>;
```

Comment:
question: does each entry define more than one pin? (`pins` is plural)

Reply:
Only one, will turn it into 'pin'.